package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "LOKOKO-CLIENT",
```

헉 요거 하나만 바꿔주세요!!!
```suggestion
  "name": "LOCOCO-CLIENT",
```
이름이 바뀌니 신경쓸게 많네요 🥹